cmd/orb_test.go

```diff
@@ -1139,7 +1139,12 @@ var _ = Describe("Orb integration tests", func() {
 				session, err := gexec.Start(command, GinkgoWriter, GinkgoWriter)
 
 				Expect(err).ShouldNot(HaveOccurred())
+				Eventually(session.Out).Should(gbytes.Say("circleci/gradle"))
+				Eventually(session.Out).Should(gbytes.Say("Jobs"))
+				Eventually(session.Out).Should(gbytes.Say("- test"))
 				Eventually(session.Out).Should(gbytes.Say("circleci/rollbar"))
+				Eventually(session.Out).Should(gbytes.Say("Commands"))
+				Eventually(session.Out).Should(gbytes.Say("- notify_deploy"))
```

Comment:
We should avoid hitting the production API in these tests

Reply:
@zzak these aren't hitting prod.  There's are two http-mock's responses set up on the lines above here:

https://github.com/CircleCI-Public/circleci-cli/blob/e16485823ec7d9b42779109bf0c549261f29bd8e/cmd/orb_test.go#L1118

and here:

https://github.com/CircleCI-Public/circleci-cli/blob/e16485823ec7d9b42779109bf0c549261f29bd8e/cmd/orb_test.go#L1134